src/analysis/retail/warlock/demonology/modules/talents/Doom.tsx

```diff
@@ -71,12 +137,14 @@ class Doom extends Analyzer {
       <Statistic
         category={STATISTIC_CATEGORY.TALENTS}
         size="flexible"
-        tooltip={`${formatThousands(this.damage)} damage`}
+        tooltip="Doom tracking - 20s debuff applied by Demonbolt when consuming Demonic Core"
```

Comment:
You should still be displaying the damage in the tooltip.
```suggestion
        tooltip={
          <>
            {formatNumber(this.doom.damage)} damage
            <br />
            Doom tracking - 20s debuff applied by Demonbolt when consuming Demonic Core
          </>
        }
```